expfactory/cli/install.py

```diff
@@ -124,7 +124,7 @@ def main(args, parser, subparser):
         with open(instruct, "w") as filey:
             filey.writelines(config["instructions"])
 
-    if not os.path.exists(dest):
+    if not os.path.exists(os.path.abspath(dest)):
         os.system("mkdir -p %s" % dest)
```

Comment:
There won't be an issue for the mkdir -p line? (and for other fixes here, for other usage of the variable in the function context?

Reply:
I'm just wondering if you've fixed the right area - eg.., install.py has some hard coded paths with a separator, which I don't think would work on windows?